packages/lib/src/components/internal/OpenInvoice/OpenInvoice.tsx

```diff
@@ -59,7 +59,7 @@ export default function OpenInvoice(props: OpenInvoiceProps) {
         fieldTypeMappingFn: mapFieldKey
     });
 
-    const specifications = useMemo(() => new Specifications(), []);
+    const specifications = useMemo(() => new Specifications(props.deliveryAddressSpecification), []);
```

Comment:
Why use `props.deliveryAddressSpecification` particularly 🤔? I would assume delivery address and billing address specifications can be different.

Reply:
This is an easy way to enhance the Billing Address specifications with with those needed by the deliveryAddress i.e. first and last name.
The other solution would be to create a separate specifications object just for DeliveryAddress. Is this what you are proposing?

Reply:
I don't think it's a problem so far, but it might get problematic when deliveryAddress and billingAddress schema differ too much, e.g (an extreme case): deliveryAddress schema contains postal code, street and country, billingAddress contains first name, last name. Then the layout would be completely wrong.

I like your first approach which decouples delivery address and billing address 💯.

Reply:
ok, I'll go with the first approach